api/tests/domain/reimbursement_test.py

```diff
@@ -413,6 +413,23 @@ def test_offerer_with_category_rule(self):
         assert finder.get_rule(booking3) is None  # outside `rule.timespan`
         assert finder.get_rule(booking4) is None  # no rule for this offerer
 
+    def test_rule_offer_rule_priority(self):
+        yesterday = datetime.utcnow() - timedelta(days=1)
+        far_in_the_past = datetime.utcnow() - timedelta(days=800)
+        yesterday_booking = bookings_factories.UsedBookingFactory()
+        offerer = yesterday_booking.offerer
+        offer = yesterday_booking.stock.offer
+        ancient_booking = bookings_factories.UsedBookingFactory(stock=yesterday_booking.stock, dateUsed=far_in_the_past)
+        another_booking = bookings_factories.UsedBookingFactory()
+        offer_rule = finance_factories.CustomReimbursementRuleFactory(offer=offer, timespan=(yesterday, None))
+        _offerer_rule = finance_factories.CustomReimbursementRuleFactory(offerer=offerer, timespan=(yesterday, None))
+
+        finder = reimbursement.CustomRuleFinder()
+
+        assert finder.get_rule(yesterday_booking) == offer_rule  # Custom rule on Offer taks over rule on Offerer
```

Comment:
(wording / question de goût) 

```suggestion
        assert finder.get_rule(yesterday_booking) == offer_rule  # Custom rule on Offer has priority over rule on Offerer
```

Reply:
question de typo aussi; merci ;)